pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -398,7 +398,7 @@ func (asw *actualStateOfWorld) addVolume(
 	}
 
 	pluginIsAttachable := false
-	if _, ok := volumePlugin.(volume.AttachableVolumePlugin); ok {
+	if thePlugin, err := asw.volumePluginMgr.FindAttachablePluginBySpec(volumeSpec); err == nil && thePlugin != nil {
```

Comment:
Nit: `thePlugin` seems like a bit of a strange variable name... if we move forward with this code, can we rethink the variable name? I think `attachablePlugin` could be more descriptive.

Reply:
good point, thanks